includes/admin/upgrades/v3/upgrade-actions.php

```diff
@@ -48,10 +48,10 @@ function edd_process_v3_upgrade() {
 
 	// If we have a step already saved, use that instead.
 	// This is commented out for now because some changes are required in the migration processes first.
```

Comment:
I'm thinking we can lose this comment now.  Or maybe change `is` to `was`?

Reply:
Whoopsies, good catch.